scripts/create-pr.sh

```
@@ -0,0 +1,34 @@
+BASE="${2:-main}"
+
+if [ "$BRANCH" = "$BASE" ]; then
+  echo "ERROR: Cannot create PR from $BASE to $BASE"
+  exit 1
+fi
+
+echo "Creating PR: $BRANCH → $BASE"
+echo "Title: $TITLE"
+
+# Get commit messages for body
+BODY=$(git log --pretty=format:"- %s" $BASE..$BRANCH | head -10)
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)